ukrdc_sqla/ukrdc.py

```diff
@@ -492,6 +494,8 @@ class RenalDiagnosis(Base):
     actioncode = Column(String(3))
     externalid = Column(String(100))
     update_date = Column(DateTime)
+    biopsyperformed = Column(String)
```

Comment:
These two also need adding to the diagnosis table.

Reply:
Based on the picklist here: 
https://github.com/renalreg/resources/blob/f9ef07806d39f89174d01b50d312cc0bcf423633/schema/ukrdc/Diagnoses/RenalDiagnosis.xsd#L12

Maybe 10 characters or so?

Reply:
Whatever's the longest value in the XML schema.

That said other than as a basic check against invalid values there isn't a storage space benefit from being parsimonious with the field sizes and you could just set them to 50/100 characters or whatever.